run_page/gpxtrackposter/year_range.py

```diff
@@ -82,5 +88,9 @@ def count(self) -> Optional[int]:
             return None
         return 1 + self.to_year - self.from_year
 
+    def count_with_data(self):
+        """Return number of years which has running data"""
+        return len(self.years_dict)
+
```

Comment:
can we just make it a property?

like

@property
def real_year():
    pass

Reply:
got it!